mudur/userprofile/static/userprofile/js/userprofile.js

```diff
@@ -39,6 +39,47 @@ function countrychanged(selectbox){
         $("#id_ykimlikno").parent().parent().find('div').first().append("<span class='required'></span>");
     }
 }
+
+function mobilephonenumberchanged(){
+    var mobile = document.getElementById('id_mobilephonenumber').value;
+    var filledMobilePhoneNumberCheckbox = document.getElementById("id_filled_mobile_phone_number");
+    var EmptyMobilePhoneNumberCheckbox = document.getElementById("id_empty_mobile_phone_number");
+
+    if (mobile.trim() === "") {
+        $("#id_filled_mobile_phone_number").parent().parent().find('span').hide();
+        $("#id_empty_mobile_phone_number").parent().parent().find('div').first().append("<span class='required'></span>");
+        filledMobilePhoneNumberCheckbox.style.display = "none";
+        EmptyMobilePhoneNumberCheckbox.style.display = "block";
+    } else {
+        filledMobilePhoneNumberCheckbox.style.display = "block";
+        EmptyMobilePhoneNumberCheckbox.style.display = "none";
+        $("#id_empty_mobile_phone_number").parent().parent().find('span').hide();
+        $("#id_filled_mobile_phone_number").parent().parent().find('div').first().append("<span class='required'></span>");
```

Comment:
javascript related parts of this app is terribly designed.  we need to return to this later. not now.